CONTRIBUTING.md

```diff
@@ -0,0 +1,100 @@
+
+In this file you can find basic information on contributing to [F´](https://github.com/nasa/fprime). We will walk through how to contribute and the process contributions follow. Remember, we may ask for changes or adjustments to make your submission the best it can be. Fear not! Your submission is still valued! You may even comment on other submissions to help them improve.
+
+## Ways of Contributing
+
+The best way to contribute to F´ is to remain positive and engaged. Just about every contribution needs some improvement before it is ready to be folded in. Stand behind your work, push it forward, and work with us!
+
+Specific Ways to Contribute:
+- [Ask a Question or Suggest Improvements](https://github.com/nasa/fprime/discussions/new)
+- [Report a Bug or Mistake](https://github.com/nasa/fprime/issues/new/choose)
+- [Review Contributions](https://github.com/nasa/fprime/pulls)
+- Submit a Pull Request
+- Contribute to Ongoing Discussions and Reviews
+
+Feel free to contribute any way that suits your skills and enjoy.
+
+## Where to Start
+
+First, contributors should build some understanding of F´. Read through the documentation, try a tutorial, or run a reference application. Contributors can find information in our [documentation](https://nasa.github.io/fprime/). Keep track of inconsistencies or bugs as these should be reported!
+
+When you are ready to join discussions and submit bug reports use one of the above links!
+
+To contribute to the F´ framework directly, consider writing [needed documentation](https://github.com/nasa/fprime/issues?q=is%3Aissue+is%3Aopen+label%3ADocumentation) or starting with an [easy first issue](https://github.com/nasa/fprime/issues?q=is%3Aissue+is%3Aopen+label%3A%22Easy+First+Issue%22). When starting to modify F´ directly, ask questions, seek help, and be patient. Remember to review the project structure, development process, and helpful tips sections below.
+
+## Project Structure
+
+The F´ project is designed as a base software [framework](https://github.com/nasa/fprime) with additional [packages](https://github.com/fprime-community) designed to extend the framework. This means that occasionally we may move contributions in or out of these packages.
+
+Key packages include:
+
+- [fpp](https://github.com/fprime-community/fpp): fpp development repository
+- [fprime-tools](https://github.com/fprime-community/fprime-tools): `fprime-util` development repository
+- [fprime-gds](https://github.com/fprime-community/fprime-gds): `fprime-gds` development repository
+
+
+### F´ Repository Structure
+
+Contributors to the [fprime](https://github.com/nasa/fprime) repository should understand the following key folders:
+
+- [docs/UsersGuide](https://github.com/nasa/fprime/tree/devel/docs/UsersGuide): add new documentation in this or a subfolder
+- [Fw](https://github.com/nasa/fprime/tree/devel/Fw): changes here will be reviewed carefully
```

Comment:
You might want to clarify why core framework changes are reviewed carefully

Reply:
Fixed